tests/aws-cpp-sdk-lambda-integration-tests/FunctionTest.cpp

```diff
@@ -247,14 +246,14 @@ class FunctionTest : public ::testing::Test {
 
         functionCode.SetZipFile(Aws::Utils::ByteBuffer((unsigned char*)buffer.str().c_str(), buffer.str().length()));
         createFunctionRequest.SetCode(functionCode);
-        createFunctionRequest.SetRuntime(Aws::Lambda::Model::Runtime::nodejs12_x);
+        createFunctionRequest.SetRuntime(Aws::Lambda::Model::Runtime::nodejs16_x);
 
         CreateFunctionOutcome createFunctionOutcome = m_client->CreateFunction(createFunctionRequest);
         AWS_ASSERT_SUCCESS(createFunctionOutcome);
         ASSERT_EQ(functionName,createFunctionOutcome.GetResult().GetFunctionName());
         ASSERT_EQ("test.handler",createFunctionOutcome.GetResult().GetHandler());
         ASSERT_EQ(roleARN,createFunctionOutcome.GetResult().GetRole());
-        ASSERT_EQ(Aws::Lambda::Model::Runtime::nodejs12_x, createFunctionOutcome.GetResult().GetRuntime());
+        ASSERT_EQ(Aws::Lambda::Model::Runtime::nodejs16_x, createFunctionOutcome.GetResult().GetRuntime());
```

Comment:
16? just do 20 or whatever is current now unless you want to update in a couple months

Reply:
good call updated